functions/bool2num_hash_recursive.pp

```diff
@@ -34,7 +34,7 @@ function yum::bool2num_hash_recursive($arg) {
       Hash    => yum::bool2num_hash_recursive($value),
       default => $value,
     }
-    Hash({ $key => $return_value })
+    Hash( { $key => $return_value })
```

Comment:
This one looks odd. Why is `Hash()` needed in he first place? I think the whole map can be removed and put in the reduce.

Reply:
I took a stab at it, but couldn't get rid of it. This does suggest a bug in the strict linting. If it at least also allowed a space at the end it would at least be symmetrical, but now it isn't.

Reply:
I raised https://github.com/kuleuven/puppet-lint-manifest_whitespace-check/issues/8 for this. Should we continue and merge this as is?. Because the PR fixes some gemfile dependencies.

Reply:
We should definitely merge it. A linting issue shouldn't hold up fixes.